### PR TITLE
replace new lsp for vlang

### PR DESCRIPTION
**Pull request by lydiandy**
replace new lsp for vlang

### CODE REVIEW

**Comment by pascalkuthe** [comment]
The pipeline is not failing on master and the repo ist will reachale for me


It seems like you are switching to a different grammar here (and different language sever) which potentially requires query changes

File: languages.toml [hunk 1]
```diff
@@ -70,7 +70,7 @@ terraform-ls = { command = "terraform-ls", args = ["serve"] }
 texlab = { command = "texlab" }
 vala-language-server = { command = "vala-language-server" }
 vhdl_ls = { command = "vhdl_ls", args = [] }
-vlang-language-server = { command = "v", args = ["ls"] }
+vlang-language-server = {command = "v-analyzer", args = [""]}
```
Comment (the-mikedavis) [inline languages.toml:73]:
```suggestion
vlang-language-server = { command = "v-analyzer" }
```

File: runtime/queries/v/highlights.scm [hunk 2]
```diff
@@ -1,83 +1,112 @@
...
+ (c_string_literal)
+ (raw_string_literal)
+ (interpreted_string_literal)
+ (string_interpolation)
+ (rune_literal)
+] @string
 
+(string_interpolation
+ (braced_interpolation_opening) @punctuation.bracket
+ (interpolated_expression) @none
```
Comment (the-mikedavis) [inline runtime/queries/v/highlights.scm:88]:
```suggestion
 (interpolated_expression) @embedded
```

**Review by the-mikedavis — COMMENTED** [review]
It looks like the textobjects.scm queries will need to be adjusted too

**Review by the-mikedavis — APPROVED** [review]

**Review by pascalkuthe — APPROVED** [review]

**Pull request merged**